WordPress/Classes/ViewRelated/Media/StockPhotos/StockPhotosDataSource.swift

```diff
@@ -176,7 +178,6 @@ extension StockPhotosDataSource: StockPhotosDataLoaderDelegate {
             appendMedia(with: media)
         }
     }
-
```

Comment:
Probably best to leave this space between functions.

Reply:
I got some lint errors when adding spaces and it prevented committing without removing them (an old issue maybe cause the branch was too behind?).